src/views/Project/index.jsx

```diff
@@ -1,111 +1,113 @@
-  },
-  {
-    project: 'Taskcluster',
-    summary: '44 - Add pagination to auth.listRoles',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-03-05',
-  },
-  {
-    project: 'Taskcluster',
-    summary:
-      '1306494 - Add a diff+commit submit button to some text areas in tc-tools',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-02-12',
-  },
-  {
-    project: 'Taskcluster',
-    summary:
-      '333 - Make azure-entities and azure-blob0-storage independent of tc-lib-monitor',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-02-04',
-  },
-  {
-    project: 'Taskcluster',
-    summary: '1344912 - Support tag events, too',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-01-01',
-  },
-  {
-    project: 'Taskcluster',
-    summary: '1453714 - Return http 424 instead of 403 for error artifacts',
-    tag: 'Python',
-    assignee: 'None',
-    lastUpdated: '2017-12-04',
-  },
-];
+const tagReposMapping = repositories =>
+  Object.keys(repositories).reduce((prev, key) => {
+    const curr = [...(prev[repositories[key]] || []), key];
+
+    return {
+      ...prev,
+      [repositories[key]]: curr,
+    };
+  }, {});
 
 @hot(module)
+@graphql(issuesQuery, {
+  skip: props => !projects[props.match.params.project].repositories,
+  options: () => ({
+    fetchPolicy: 'network-only',
+    variables: {
+      searchQuery: '',
+    },
+  }),
+})
 export default class Project extends Component {
+  state = {
+    loading: true,
+  };
+
+  componentDidMount() {
+    this.load();
+  }
+
+  fetch = searchQuery => {
+    const {
+      data: { fetchMore },
+    } = this.props;
+
+    return fetchMore({
+      query: issuesQuery,
+      variables: {
+        searchQuery,
+      },
+      updateQuery(previousResult, { fetchMoreResult }) {
+        const moreNodes = fetchMoreResult.search.nodes;
+
+        if (!moreNodes.length) {
+          return previousResult;
+        }
+
+        return dotProp.set(
+          previousResult,
+          'search.nodes',
+          moreNodes.concat(previousResult.search.nodes)
+        );
+      },
+    });
+  };
+
+  load = async () => {
+    const project = projects[this.props.match.params.project];
+    const repositories = mergeAll(project.repositories);
+    const tagsMapping = tagReposMapping(repositories);
+
+    await Promise.all(
+      Object.entries(tagsMapping).map(([tag, repos]) => {
+        const searchQuery = [
+          repos.map(repo => `repo:${repo}`).join(' '),
+          `label:${tag}`,
+          'state:open',
+        ].join(' ');
+
+        return this.fetch(searchQuery);
+      })
+    );
+
+    this.setState({ loading: false });
+  };
+
   render() {
+    const { data } = this.props;
+    const search = (data && data.search) || { nodes: [] };
+    const error = (data && data.error) || undefined;
+    const { loading } = this.state;
     const project = projects[this.props.match.params.project];
+    const issues =
+      search &&
```

Comment:
`const issues = data && data.search && uniqBy(...); `

Reply:
how about `const issues = (data && data.search && uniqBy(...)) || []; ` so that in case there is no fetching from github, it will still render the table?

Reply:
Yep, that works.